tests/system/Database/Live/OCI8/GetFieldDataTestCase.php

```diff
@@ -58,7 +58,7 @@ public function testGetFieldDataDefault(): void
         $fields = $this->db->getFieldData($this->table);
 
         $idDefault = $this->getFieldMetaData('id', $this->table)->default;
-        $this->assertMatchesRegularExpression('/"ORACLE"."ISEQ\$\$_[0-9]+".nextval/', $idDefault);
+        $this->assertMatchesRegularExpression('/"ORACLE"."ISEQ\$\$_\d+".nextval/', $idDefault);
```

Comment:
`\d` means a digit, but `[0-9]` means `0` to `9` (ASCII Arabic number). They are not the same.
Without `u` option, the result may be the exactly same, but the code implies different things.
So I don' t like this refactoring.

Reply:
It doesn't have `/u` so should be safe, also, this is test so not apply to end user